static/src/javascripts/projects/common/modules/ui/message.spec.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import $ from 'lib/$';
+import _userPrefs from 'common/modules/user-prefs';
+import { Message } from './message';
+
+jest.mock('common/modules/user-prefs', () => ({
+    get: jest.fn(),
+    set: jest.fn(),
+}));
+
+jest.mock('common/modules/analytics/register', () => ({
+    begin: jest.fn(),
+}));
+
+const userPrefs: any = _userPrefs;
+
+describe('Message', () => {
+    beforeEach(() => {
+        if (document.body) {
+            document.body.innerHTML = `
+                <div id="header" class="l-header"></div>
+                <div class="js-site-message is-hidden">
+                    <div class="js-site-message-copy">...</div>
+                    <button class="site-message__close"></button>
+                </div>
+                <div class="site-message--footer is-hidden js-footer-message">
+                    <div class="site-message__copy js-footer-site-message-copy u-cf"></div>
+                </div>'
+            `;
+        }
+    });
+
+    it('Show a message', () => {
+        new Message('foo').show('hello world');
+        expect($('.js-site-message-copy').text()).toContain('hello world');
+        expect($('.js-footer-site-message-copy').text()).not.toContain(
+            'hello world'
+        );
+        expect($('.js-site-message').hasClass('is-hidden')).toBeFalsy();
```

Comment:
Again for strictness, can we use `toEqual(false)` / `toEqual(true)` instead of `toBeFalsy()` / `toBeTruthy()`?